src/Components/DraggableResizer/DraggableResizer.tsx

```diff
@@ -162,16 +193,21 @@ export const DraggableResizerRoot: FunctionComponent<DraggableResizerProps> = ({
         if (child.type !== DraggableResizerPanel) {
           return null
         }
-
         const isLastPanel = i === panelsCount - 1
         const dragging = i === dragIndex
 
+        const isCollapsibleToLower = child.props.isCollapsible
+        const isCollapsibleToUpper =
+          !isLastPanel && children && children[i + 1].props.isCollapsible
+
         return (
           <>
             {cloneElement(child, {sizePercent: calculatePanelSize(i)})}
             {!isLastPanel && (
               <DraggableResizerHandle
-                position={child.props.position}
+                isCollapsibleToUpper={isCollapsibleToLower}
+                isCollapsibleToLower={isCollapsibleToUpper}
```

Comment:
I'm curious why the `isCollapsibleToUpper` takes in `isCollapsibleToLower` value?

Reply:
That's cause I got em mixed up hah